spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/rxjava/SleuthRxJavaSchedulersHookTests.java

```diff
@@ -59,6 +73,37 @@ public void should_wrap_delegates_action_in_wrapped_action_when_delegate_is_pres
 		then(caller.toString()).isEqualTo("called_from_schedulers_hook");
 	}
 
+	@Test
+	public void should_not_create_a_span_when_current_thread_should_be_ignored()
+			throws ExecutionException, InterruptedException {
+		String threadNameToIgnore = "^MyCustomThread.*$";
```

Comment:
hard to tell which of "HystixMetricPoller" and ^^ are being tested. are both needed?